src/styles/blocks/productCard/product.scss

```diff
@@ -0,0 +1,113 @@
+.product {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  transition: transform 300ms;
+
+  &:hover{
+    transform: scale(1.2);
+  }
+
+  &__picture {
+    background-image: url(../images/imac.jpeg);
+    background-size: $imageWidth $imageHeight;
+    width: $imageWidth;
+    height: $imageHeight;
```

Comment:
The variables `$imageWidth` and `$imageHeight` are used here, but they are not defined within this file. Ensure these variables are defined elsewhere or imported correctly to avoid compilation errors.